<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Employ√©s</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tab">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>941</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_cruds">
     <attribute name="title">
      <string>Ajouter Employee</string>
     </attribute>
     <widget class="QGroupBox" name="Ajoutmodification">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>391</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajout</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_Nom">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Prenom">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>230</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Function</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Phone</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>330</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Mdp</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Phone">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>180</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Function">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>280</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Email">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>230</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Mdp">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>330</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_CIN">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_modifier">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>270</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="suppression">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>380</y>
        <width>281</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Suppression</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Supp">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supprimer">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>100</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="affichage">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>20</y>
        <width>461</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Affichage</string>
      </property>
      <widget class="QTableView" name="tab_employee">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>311</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tri">
     <attribute name="title">
      <string>Chercher Employer</string>
     </attribute>
     <widget class="QGroupBox" name="recherche_tri">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>671</width>
        <height>501</height>
       </rect>
      </property>
      <property name="title">
       <string>Recherche</string>
      </property>
      <widget class="QPushButton" name="pb_chercher">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>440</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QTableView" name="tab_employee_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>441</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_rechercher">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>50</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>rechercher</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_tri">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>440</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="tri_CIN">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="tri_nom">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="tri_prenom">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="tri_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri par :</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>agenda</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1096</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGestion_des_employ_es">
    <property name="title">
     <string>Gestion des employ√©es</string>
    </property>
   </widget>
   <addaction name="menuGestion_des_employ_es"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
